bower.json

```diff
@@ -12,17 +12,17 @@
     "output"
   ],
   "dependencies": {
-    "purescript-arrays": "^4.4.0",
-    "purescript-bigints": "^3.5.0",
-    "purescript-integers": "^3.2.0",
-    "purescript-maybe": "^3.1.0",
-    "purescript-partial": "^1.2.0",
-    "purescript-prelude": "^3.2.0",
-    "purescript-strings": "^3.5.0"
+    "purescript-arrays": "^5.0.0",
+    "purescript-bigints": "^4.0.0",
+    "purescript-integers": "^4.0.0",
+    "purescript-maybe": "^4.0.0",
+    "purescript-partial": "^2.0.0",
+    "purescript-prelude": "^4.0.1",
+    "purescript-strings": "^4.0.0"
   },
   "devDependencies": {
-    "purescript-psci-support": "^3.0.0",
-    "purescript-spec": "^2.0.0",
-    "purescript-spec-discovery": "^2.0.0"
+    "purescript-psci-support": "^4.0.0",
+    "purescript-spec": "^3.0.0",
+    "purescript-spec-discovery": "paulyoung/purescript-spec-discovery#paulyoung/0.12"
```

Comment:
This is waiting on https://github.com/owickstrom/purescript-spec-discovery/pull/7